reactor-core/src/main/java/reactor/core/publisher/FluxIdentityProcessor.java

```diff
@@ -1,26 +0,0 @@
-/*
- * Copyright (c) 2011-Present VMware Inc. or its affiliates, All Rights Reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *        https://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package reactor.core.publisher;
-
-/**
- * A {@link FluxProcessor} that has the same input and output types.
- * 
- * @author Simon Baslé
- */
-public abstract class FluxIdentityProcessor<T> extends FluxProcessor<T, T> {
```

Comment:
the rationale for adding this was that `FluxProcessor<String, String>` feels clunky, and was such a pain to migrate when removing deprecation warnings in the test code. Maybe that pain will be a good motivator for people to move to `Sink`? But maybe on the contrary it will just make them angry 😝

Reply:
Yeah the rationale is to not use processors api at all in the future so no need to spend any more surface api for them.